cunumeric/module.py

```diff
@@ -4321,7 +4490,7 @@ def unique(
     `axis` is also not handled currently.
 
     """
-    if any((return_index, return_inverse, return_counts, axis)):
+    if _builtin_any((return_index, return_inverse, return_counts, axis)):
```

Comment:
I believe this was a bug

Reply:
nice catch. this should have been renamed when `any` was added to the API, but somehow was overlooked.